README.md

```diff
@@ -56,12 +63,12 @@ The following commands install and run the development version of The Lounge:
 ```sh
 git clone https://github.com/thelounge/lounge.git
 cd lounge
-npm install
-NODE_ENV=production npm run build
-npm start
+yarn install
+NODE_ENV=production yarn build
+yarn start
 ```
 
-When installed like this, npm doesn't create a `thelounge` executable. Use `npm start -- <command>` to run subcommands.
+When installed like this, npm doesn't create a `thelounge` executable. Use `yarn start -- <command>` or `node index.js <command>` to run subcommands.
```

Comment:
With `yarn`, commands can be run with `yarn start <command>` (i.e. no need for `--`).

Reply:
Shouldn't this read `yarn doesn't create`?

Reply:
`yarn start command` didn't work anyway because `start` already runs a command. I reworded it.

Reply:
> `start` already runs a command.

What did you mean by this?

Reply:
It would turn into `node index.js start <command>` so it didn't exactly do what was implied.

Reply:
Damn, didn't realize this has been broken for a year 😂 https://github.com/thelounge/lounge/commit/86ed0b6e59f1e53b9967f3344b4dd68de0cd6ca2#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L16